src/style.css

```
@@ -1,3 +1,73 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  background-color: #fff;
+}
+
+@media (min-width: 1024px) and (max-width: 1200px) {
+  header {
+    height: 60px;
+  }
+}
+
+header {
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: baseline;
+  position: relative;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  align-content: stretch;
+  flex-direction: row;
+  margin-right: 50px;
+}
+
+li {
+  margin: 20px;
+  align-items: center;
+  text-align: center;
+  box-sizing: border-box;
+}
+
+.is-active {
+  color: #00acdc;
+  text-decoration: none;
+}
+
+.is-active::after {
+  position: relative;
+  width: 37px;
```

```suggestion
  width: 100%;
```
different links will be have different width